modules/search-service-impl/src/main/java/org/opencastproject/search/endpoint/SearchRestService.java

```diff
@@ -465,35 +457,32 @@ public Response getEpisode(
 
     boolean invalidSeries = false;
     if (seriesName != null) {
-      if (seriesService == null) {
-        return Response.status(Response.Status.NOT_IMPLEMENTED).build();
-      }
-      DublinCoreCatalogList result;
+      SearchResult result = new SearchResultImpl();
       try {
-        result = seriesService.getSeries(new SeriesQuery().setSeriesTitle(seriesName));
-      } catch (SeriesException e) {
+        SearchQuery seriesSearch = new SearchQuery();
+        seriesSearch.includeSeries(true)
+            .includeEpisodes(false)
+            .withQuery("*:(dc_title___:" + seriesName + ")");
```

Comment:
I know I complain too much 🙈
…but this poses a security issue since you put unfiltered user input into a raw query. A user could easily break out by submitting a series name like `xy ) )` and then continue the query with whatever they want, e.g. circumventing all ACL checks.

I haven't tried it by there is already a method to escape input and I hope adding this will fix the issue:
```java
seriesName = SolrUtils.clean(seriesName);
```

Reply:
No problem, it's a valid point. I changed it to use the clean function, and still works as expected.